### ISSUE TITLE
Check package license and reduce size

### ISSUE THREAD

**Issue by wleoncio** [SystemsBiologist/CellMigRation#29]
Source: https://github.com/Bioconductor/Contributions/issues/1618#issuecomment-696258765

- [x] NOTE: checking installed package size. Installed size is 8.2Mb, data
directory is 6.6Mb.
- [x] REQUIRED: Since you are using a standard license you do not need to
include the LICENSE file in your package.

**Comment by wleoncio** [comment]
@SystemsBiologist, @dami82, I am trying to reduce the size of the `data` folder, which is composed of the following elements:

```
total 3.0M
-rw-rw-r-- 1 waldir waldir 869K Mar 25  2020 TrackCellsDataset.rda
-rw-rw-r-- 1 waldir waldir 188K Mar 25  2020 TrajectoryDataset.rda
-rw-rw-r-- 1 waldir waldir 1.9M Sep 30 07:26 VignBuilderDataset.rda
-rw-rw-r-- 1 waldir waldir  50K Mar 25  2020 WSADataset.rda
```
Do we need all these objects? The largest one is, by far, VignBuilderDataset.rda. I tried resaving it with max compression but it didn't help, but maybe some of its elements can be dropped. What do you say?

**Comment by wleoncio** [comment]
Fixed on f38498fdda3554d5ef33613d1b5ee4970b94bdc2.
